### DISCUSSION TITLE
default param values for sacrebleu

### DISCUSSION THREAD

**Discussion by base-y** [huggingface/datasets#4289] [Show and tell]
I see default params for sacrebleu as `smooth_method=exp` and tokenize=None (uses 13a default tokenizer) while in `sacrebleu` python lib (https://github.com/mjpost/sacrebleu), the default params seem to be `smooth_method=floor` and `tokenize='none'` (which uses BaseTokenizer).

Which set is officially recognized and why was exp smooth method chosen in `hf datasets`?

**Comment by mariosasko** [comment]
Hi! The default parameters should match since our implementation calls `sacrebleu.corpus_blue`, which has the following signature: https://github.com/mjpost/sacrebleu/blob/8e7abf591f6acff69dcbcad8d7af94754296fcb4/sacrebleu/compat.py#L9-L16 (notice `smooth-methd='exp'` and `tokenize=BLEU.TOKENIZER_DEFAULT`). And if you follow the code further, you can see that `tokenize=None` is equal to `tokenize=BLEU.TOKENIZER_DEFAULT`.
